microprofile/metrics-smallrye/src/main/resources/io/smallrye/metrics/base-metrics.properties

```diff
@@ -83,15 +80,15 @@ memory.usedNonHeap.unit: bytes
 thread.count.mbean: java.lang:type=Threading/ThreadCount
 thread.count.description: Number of currently deployed threads
 thread.count.unit: none
-thread.count.type: counter
+thread.count.type: gauge
```

Comment:
Again here and for the rest of the file, I guess the replacement of 'counter' with 'gauge' is intentional?

Reply:
yes, it is intentional to comply with the spec https://github.com/eclipse/microprofile-metrics/blob/master/spec/src/main/asciidoc/required-metrics.adoc#thread-jvm-stats

Reply:
likewise for `thread.daemon.count.type` and `thread.max.count.type` below